filesystems/blob/src/main/scala/cromwell/filesystems/blob/BlobPathBuilder.scala

```diff
@@ -45,7 +47,7 @@ object BlobPathBuilder {
       val uri = parseURI(string)
       val storageAccount = parseStorageAccount(parseURI(endpoint))
       val hasContainer = uri.getPath().split("/").filter(!_.isEmpty()).headOption.contains(container)
-      def hasEndpoint = parseStorageAccount(uri).contains(storageAccount.get)
+      def hasEndpoint = storageAccount.map(parseStorageAccount(uri).contains(_)).getOrElse(false)
```

Comment:
Can make this even better with `storageAccount.exists(parseStorageAccount(uri).contains(_))`